spec/integ/megolm-integ.spec.ts

```diff
@@ -327,7 +330,9 @@ describe("megolm", () => {
         const room = aliceTestClient.client.getRoom(ROOM_ID)!;
         const event = room.getLiveTimeline().getEvents()[0];
         expect(event.isEncrypted()).toBe(true);
-        const decryptedEvent = await testUtils.awaitDecryption(event);
+
+        // it probably won't be decrypted yet, because it takes a while to process the olm keys
+        const decryptedEvent = await testUtils.awaitDecryption(event, { waitOnDecryptionFailure: true });
```

Comment:
this is needed due to timing differences in processing `/sync` responses: there are fewer `await`s, which means that we finish processing the `/sync` while there are still to-device messages in the queue for processing.